### PR TITLE
Fix cron.php error

### PR DESCRIPTION
**Pull request by DavidePrincipi**
Allow to remove 6.10 from the "stable_releases" array

### CODE REVIEW

File: centos.php [hunk 1]
```diff
@@ -34,8 +34,9 @@
 $arch = $_GET['arch'];
 $repo = $_GET['repo'];
 
+$known_releases = array_unique(array_merge($stable_releases, $vault_releases, $development_releases));
 $valid_release = $release == '6';
```
Comment (gsanchietti) [inline centos.php:38]: Is it better to change the default to 7?

File: config.php [hunk 2]
```diff
@@ -49,13 +38,13 @@
     '7.8.2003',
 );
 
+
```
Comment (gsanchietti) [inline config.php:41]: If possible, clean the patch and remove the whiteline

File: config.php [hunk 3]
```diff
@@ -70,3 +59,16 @@
     'armhfp',
     'aarch64',
 );
+
+///
+/// Select CentOS mirrors from specific countries across the globe
+///
```
Comment (gsanchietti) [inline config.php:65]: There 3 `/`, I suspect the editor added an extra `/`

File: centos.php (hunk 1 above)
Comment (DavidePrincipi) [reply centos.php:38]: The centos.php script is used **only** by version 6. Changing the script is not strictly required here, but I'd prefer to use the same variable "known_releases" and keep it aligned to nethserver.php.

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**